src/Symfony/Component/Validator/Constraints/PasswordStrength.php

```diff
@@ -45,12 +43,10 @@ final class PasswordStrength extends Constraint
      */
     public array $restrictedData = [];
 
+    public null|\Closure $passwordStrengthEstimator = null;
```

Comment:
Requiring a closure in the constraint is a no-go: those constraints are usable as annotations, which get serialized in the cache. And the validation mapping also gets serialized in the symfony/validator metadata cache.

And I doubt you will want different estimators in different places in the project. So this looks like something that should be injected in the ConstraintValidator instead (being configurable through the FrameworkBundle configuration)

Reply:
That was my proposal in another thread. I created an interface that is now injected in the constructor.
I will also create a configuration option for the bundle